pureconfig/src/main/scala/com/avast/sst/pureconfig/implicits/Http4sBlazeServer.scala

```diff
@@ -0,0 +1,14 @@
+package com.avast.sst.pureconfig.implicits
```

Comment:
we probably want to use the trick/technique from cats (& co.) where `implicits` is an object which has all implicit classes/convertors/... as members, so that you can then
```scala
import com.avast.sst.pureconfig.implicits._
```
and all implicit things are in scope

Reply:
Not in this case. Because the dependencies are optional you don't want to import everything at once. It would work if they were defined as `lazy val` but I'd rather force the user to make an explicit decision than to rely on this.